### PR TITLE
ADF-835 Add assign value to calendar cypress command

### PR DESCRIPTION
**Pull request by meefox**
Related to https://oat-sa.atlassian.net/browse/ADF-835

Complimentary PR for https://github.com/oat-sa/extension-tao-item/pull/571

### CODE REVIEW

File: views/cypress/support/resourceProperties.js [hunk 1]
```diff
@@ -140,6 +140,17 @@ Cypress.Commands.add('assignValueToTextProperty', (property, value) => {
     cy.get(`[data-testid="${property.name}"]`).clear().type(value);
 });
 
+/**
+ * Assigns value to the calendar property
+ * @param {Object} property - property to assign value to
+ * @param {String} property.name - name of the property
+ * @param {String} value - value to assign
+*/
+Cypress.Commands.add('assignValueToCalendarProperty', (property, value) => {
```
Comment (gabrielfs7) [inline views/cypress/support/resourceProperties.js:149]: What about the password property (and the other properties)?

Comment (gabrielfs7) [reply views/cypress/support/resourceProperties.js:149]: https://github.com/oat-sa/extension-tao-item/pull/571/files#r811649620

**Review by Silvia-loza — APPROVED** [review]
It LGTM. 
- [X] New code is covered by tests (if applicable)
- [X] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X]  Pull request's target is not` master`
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

File: views/cypress/support/resourceProperties.js (hunk 1 above)
Comment (meefox) [reply views/cypress/support/resourceProperties.js:149]: it is not necessary for now, the password is the same in terms of adding value

Comment (gabrielfs7) [reply views/cypress/support/resourceProperties.js:149]: But why creating the calendar here and not the password? What if someone wants to validate password property in other test, should they repeat the code you wrote the in the tao-item extension?

**Review by bziondik — CHANGES_REQUESTED** [review]
Failed because of expired token issue, after  tests `Main Item Class creation and editing`
image

**Review by gabrielfs7 — APPROVED** [review]

**Pull request merged**